flink-connectors/flink-connector-kafka-0.10/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 
 	<!-- Allow users to pass custom connector versions -->
 	<properties>
-		<kafka.version>0.10.0.1</kafka.version>
+		<kafka.version>0.10.1.0</kafka.version>
```

Comment:
Just to be sure: were there any additional dependencies as a result to this bump?